transform/format.go

```diff
@@ -121,6 +158,12 @@ func DefaultFormatValue(attr, path string, info os.FileInfo) interface{} {
 		return info.Size()
 	case "time":
 		return info.ModTime().Format(time.Stamp)
+	case "hash":
+		v, err := hash(info, path, crypto.SHA1)
+		if err != nil {
+			panic(err.Error())
```

Comment:
That's what I was thinking as well, I suppose we can let the caller handle the error?